README.md

```diff
@@ -137,6 +137,25 @@ Alternatively, you can also post your question on [Gitter](https://app.gitter.im
 
 --------------
 
+## Running locally
+
+This project uses the newer `.slnx` (Solution Filter) format instead of the traditional `.sln` file. You can open it using **Visual Studio 2019 (v16.5 or newer)** or **Visual Studio 2022**.
```

Comment:
```suggestion
This project uses the newer `.slnx` format instead of the traditional `.sln` file. You can open it using **Visual Studio 2019 (v16.5 or newer)** or **Visual Studio 2022**.
```

`.slnx` is different from `.slnf` solutions filters, that are JSON files and serve a different purpose.

Reply:
BTW, are you sure it's compatible with VS 2019, which is a fairly old beast? (I'm not even sure it supports the .NET 9.0 SDK)

Reply:
just tested it with 2019 and it does not support it. I removed it.